large-file-uploader-app/assets/iframe.html

```diff
@@ -17,11 +17,18 @@
 					}
 				);
 
-				client.get('ticket.id').then(
+				client.get('ticket.customField:custom_field_360013377572').then(
 					function(data) {
-						var element = document.getElementById('largeFileUploader');
+						var accountEntryId = data['ticket.customField:custom_field_360013377572']
 
-						element.innerHTML = '<a href="https://customer.liferay.com/project-details?p_p_id=com_liferay_osb_customer_account_entry_details_web_AccountEntryDetailsPortlet&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_osb_customer_account_entry_details_web_AccountEntryDetailsPortlet_mvcRenderCommandName=%2Fadd_ticket_attachment&_com_liferay_osb_customer_account_entry_details_web_AccountEntryDetailsPortlet_zendeskTicketId=' + data['ticket.id'] + '" target="_blank">Add Large Files over 20 MB here</a>';
+						if( accountEntryId !== null && accountEntryId !== '' ) {
```

Comment:
I would probably rewrite this as `if(accountEntryId && accountEntryId !== '') {...}`

Reply:
Ohh you are right, that will also check for `undefined` and for 0. Thank you!